pkg/operator/staticpod/controller/installer/installer_controller_test.go

```diff
@@ -1127,6 +1127,8 @@ func TestCreateInstallerPodMultiNode(t *testing.T) {
 					NodeName:           "test-node-1",
 					CurrentRevision:    1,
 					LastFailedRevision: 2,
+					LastFailedCount:    1,
```

Comment:
we didn't have `lastFailedCount` in 4.6 and older. But every cluster has been upgraded many times since then, so we can expect to have it.

Reply:
Compare the added test below which is the same as this (modified) one, but without count and revision. So this is NOT a test change, but I am extending coverage.